packages/cadl-rlc-emitter/src/index.ts

```diff
@@ -42,6 +44,11 @@ async function emitClientDefinition(rlcModels: RLCModel, program: Program) {
   await emitFile(clientDefinitionsFile, program);
 }
 
+async function emitResponseTypes(rlcModels: RLCModel, program: Program) {
+  const responsesFile = buildResponseTypes(rlcModels);
+  if (responsesFile) await emitFile(responsesFile, program);
```

Comment:
nit: it is preferred in general to add the `{}` even if it is a single line statement in the if block